slices/tar.yaml

```diff
@@ -0,0 +1,27 @@
+package: tar
```

Comment:
FYI: there is another tar slice on its way. https://github.com/canonical/chisel-releases/pull/320/files#diff-d2ba6fca2d17b4a0cebd482195bfd53d541b151be0ad2fd14496adf5e6537685